libr/bin/p/bin_sms.c

```diff
@@ -103,7 +103,7 @@ RBinPlugin r_bin_plugin_sms = {
 	.name = "sms",
 	.desc = "SEGA MasterSystem/GameGear",
 	.license = "LGPL3",
-	.load_bytes = &load_bytes,
+	.load_buffer = &load_buffer,
```

Comment:
the next step could be to do load_buffer -> load and check_buffer -> check :?

Reply:
yeah it seems reasonable.

Reply:
let's wait maybe a bit, just to not cause confusion